bedms/attr_standardizer.py

```diff
@@ -79,18 +93,54 @@ def _get_parameters(self) -> Tuple[int, int, int, int, int, float]:
                 OUTPUT_SIZE_BEDBASE,
                 DROPOUT_PROB,
             )
-        else:
-            raise ValueError(
-                f"Schema not available: {self.schema}. Presently, three schemas are available: ENCODE , FAIRTRACKS, BEDBASE"
+    
+        raise ValueError(
+                f"Schema not available: {self.schema}."
+                "Presently, three schemas are available: ENCODE , FAIRTRACKS, BEDBASE"
             )
 
-    def _load_model(self) -> nn.Module:
+    def _load_model(self) -> tuple[nn.Module, object, object]:
         """
+<<<<<<< HEAD:bedms/attr_standardizer.py
+        Calls function to load the model from HuggingFace repository
+          load vectorizer and label encoder and sets to eval().
+=======
         Calls function to load the model from HuggingFace repository and sets to eval().
-
+>>>>>>> b13f0a3f927e52b5ccb9ec6f36064f19a1f5af9e:attribute_standardizer/attr_standardizer.py
```

Comment:
Yeah this needs to be fixed... Looks like we committed and pushed mid-merge

Reply:
Yes, updated that!